internal/engine/filter/diff.go

```diff
@@ -64,6 +64,7 @@ func (f FiltersDiff) String() string {
 		FromFile: "Current",
 		ToFile:   "TO BE APPLIED",
 		Context:  f.ContextLines,
+		Color:    true,
```

Comment:
Do you mind making this optional through a flag?

Some people rely on this in scripts, which would suddenly have garbled logs in them.

To make this nicer, there could be an automatic TTY detection, but this can definitely happen later.

Reply:
Sorry for the delay in replying. The martinohmann difflib actually does automatic TTY detection already. Would you still like me to add a flag?

Reply:
I think it would still be nice to have the flag, as e.g. `grep` does:

```
--color[=WHEN]
    WHEN is never, always, or auto.
```

Reply:
Done.